<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#label{
    font:40px; 
    font-family: 幼圆;
}
</string>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>800</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>480</width>
     <height>800</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>480</width>
     <height>800</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
background-color : rgb(244, 244, 244);
}</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>291</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit_username">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>45</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
border:1px solid rgb(244, 244, 244);
border-bottom-color: gray;
background-color:rgb(244, 244, 244);
font:15px; 
font-family: Arial;
}
</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_pwd">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>45</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
border:1px solid rgb(244, 244, 244);
border-bottom-color: gray;
background-color:rgb(244, 244, 244);
font:15px; 
font-family: Arial;
}
</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_mail">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>45</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
border:1px solid rgb(244, 244, 244);
border-bottom-color: gray;
background-color:rgb(244, 244, 244);
font:15px; 
font-family: Arial;
}
</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>700</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>-1</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius:15px;
border-color:rgb(42, 180, 255);
background-color:rgb(42, 180, 255);
color:white;
font:20px; 
font-family: 幼圆;
}
QPushButton:hover{
background-color: rgb(181, 225, 250);
}</string>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
   </widget>
  </widget>
  <zorder>widget</zorder>
  <zorder>label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
